sass/layouts/services.scss

```diff
@@ -0,0 +1,45 @@
+.services {
+  background-color: $color-services-background;
+  border-bottom: 6px solid $color-services-shadow;
+  position: relative;
+  top: 66px;
```

Comment:
Why do you use such a style for top distance? Maybe padding would be more appropriate in such cases?

Reply:
Each section is pushed aside because at the top the "header" block displays: fixed;